### DISCUSSION TITLE
twinBASIC BETA 57

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#924] [Releases]
- improved: IDE control rendering, allowing many more ActiveX controls to render properly inside the IDE [ https://github.com/WaynePhillipsEA/twinbasic/issues/903#issuecomment-1159370978 ]
- fixed: BackColor now effective for controls with VisualStyles set
- fixed: issue with UserControls and Implements-Via feature
- fixed: Sample4: MyVBEAddIn regressions causing compilation errors

<hr /><em>This discussion was created from the release <a href='https://github.com/WaynePhillipsEA/twinbasic/releases/tag/beta-57'>twinBASIC BETA 57</a>.</em>

**Comment by wopag** [comment]
Brilliant! much better - only a few now just displaying blank.

**Reply by WaynePhillipsEA** [reply]
Cool.  Which ones are blank?

**Reply by wopag** [reply]
Seems its all the ones that in VB6 just display the ICON for the component (as they have no form presence)

formextender
popupcontrol
commondialog
taskdialog
trayicon

..... all looking really good !!!!

**Comment by VB6-Programming** [comment]
I couldn't get the (Custom) or (About) options (... button) to work on the control I tried.  They worked in an earlier version.

***CANCEL THIS.***

They did work but appeared behind the IDE.
They were each opened multiple times out of sight,VB6 will only allow them to be opened once.

**Reply by WaynePhillipsEA** [reply]
Thanks, I will check this.

**Reply by VB6-Programming** [reply]
Beta 58 seems to have largely fixed this.  Making them modal stops the multiple opening, and if they do open behind the IDE (I still get this, possibly a 'first-time through' issue?) you now get a message if you try to do anything else.
